nym-mix-sim/README.md

```
@@ -0,0 +1,162 @@
+# 3. In a separate terminal, send a message between the two clients
+cargo run --bin mix-client -- --src 6 --dst 7
+# Then type a message and press ENTER
+```
+
+## Commands
+
+### `init-topology`
+
+Generates a `topology.json` file describing nodes and clients.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)